### DISCUSSION TITLE
Alternate data store

### DISCUSSION THREAD

**Discussion by jeffkararo** [oqtane/oqtane.framework#883] [Q&A]
When installing oqtane for the first time, the data store set up is executed. Is it possible to pick the data store you want to use? I would like to use a noSql data store like Cosmos or RavenDB.

If it is not possible now, is that in the works as a future enhancement?

**Comment by hishamco** [comment]
Oqtane is using EF Core, so any database support EF should work, but I think there are some few places many need changes

**Comment by sbwalker** [comment]
@jeffkararo other databases are not currently supported - it is on the roadmap. Although the framework uses EF Core exclusively it is not using migrations during installation due to some challenges with supporting multi-tenancy.

**Comment by amdav** [comment]
Awesome Shaun - thanks for clarifying.  Also I am not saying this should be a priority, just a nice to have.
